src/Console/Command/DescribeCommand.php

```diff
@@ -84,7 +84,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             throw new \InvalidArgumentException(sprintf(
                 '%s %s not found.%s',
-                ucfirst($e->getType()), $name, null === $alternative ? '' : ' Did you mean "'.$alternative.'"?'
+                ucfirst($e->getType()),
+                $name,
+                null === $alternative ? '' : ' Did you mean "'.$alternative.'"?'
```

Comment:
I'm surprised there aren't more of there TBH. I apply this exception in my projects as well,
for example
```php
<?php
$a = sprintf(
    '%d %d %d %d %d %d %d %d', 
    $a, $a, $a, $a, $a, $a, $a, $a
);
```

vs.

```php
<?php
$a = sprintf(
    '%d %d %d %d %d %d %d %d', 
    $a,
    $a, 
    $a, 
    $a,
    $a, 
    $a, 
    $a, 
    $a
);
```